src/packs/advancements/skyvoid_standard/beet.yaml

```diff
@@ -0,0 +1,10 @@
+id: 'skyvoid_advancements_standard'
+name: 'SkyBlock Advancements'
+author: 'EpyonProjects'
+version: 2.0.0
```

Comment:
I'm assuming there is a major change since you changed the root advancement? Don't forget to change the load score in the load.mcfunction to match the major version (I can't comment on that file cuz it was moved and without changes).

Reply:
Hmm, no there wasn't exactly a reason for the major change other than that the pipeline was different.  I have vague memories of having this conversation already, where it shouldn't matter because it's the same output / non-breaking for the player.